flash_examples/predict/style_transfer.py

```diff
@@ -0,0 +1,23 @@
+import sys
+
+import flash
+from flash.data.utils import download_data
+from flash.utils.imports import _PYSTICHE_AVAILABLE
+from flash.vision.style_transfer import StyleTransfer
+
+if _PYSTICHE_AVAILABLE:
+    import pystiche.demo
+else:
+    print("Please, run `pip install pystiche`")
+    sys.exit(0)
+
+download_data("http://images.cocodataset.org/zips/train2014.zip", "data")
+
+data_module = ImageUnsupervisedData.from_folder("data")
+
+style_image = pystiche.demo.images()["paint"].read(size=256, edge="long")
+
+model = StyleTransfer(style_image)
+
+trainer = flash.Trainer(max_epochs=2)
```

Comment:
I also need `batch_size=4` but IDK where to put it. It doesn't seem to belong to the trainer.

Reply:
Flash Trainer sub-classes PL-Trainer which has `batch_size` as a member variable

Reply:
There is no mention of a `batch_size` in the [docs](https://pytorch-lightning.readthedocs.io/en/stable/api/pytorch_lightning.trainer.trainer.html#trainer).

Reply:
`batch_size` is passed to the `DataModule` in the `from_*` method, not the trainer. So you should be able to do: `from_folders(..., batch_size=4)` :smiley: